Amethyst/Categories/SIWindow+Amethyst.swift

```diff
@@ -11,32 +11,48 @@ import Foundation
 import Silica
 
 extension SIWindow {
-    static func topWindowForScreenAtPoint(_ point: CGPoint, withWindows windows: [SIWindow]) -> SIWindow? {
-        guard let windowDescriptions = windowDescriptions(.optionOnScreenOnly, windowID: CGWindowID(0)), windowDescriptions.count > 0 else {
-            return nil
-        }
+    fileprivate static func windowInformation(_ windows: [SIWindow]) -> (IDs: Set<CGWindowID>, descriptions: [[String: AnyObject]]?) {
+        let ids = Set(windows.map { $0.windowID() })
+        return (IDs: ids, descriptions: windowDescriptions(.optionOnScreenOnly, windowID: CGWindowID(0)))
```

Comment:
`CGWindowID(0)` was in the original code, likely because it eventually gets passed as `windowID: CGWindowID` to `CGWindowListCopyWindowInfo` (not as an optional).  I'm not sure how to fix this, and the fix itself feels outside the scope of this PR.

Reply:
Did some more [research](https://stackoverflow.com/a/30337008/2063546) on this.  Apparently the `CGWindowID(0)` is how they do it in CGWindow.h:

```c
/* A value which is never the window ID of any window. */

#define kCGNullWindowID ((CGWindowID)0) 
```

I think the `#define` keeps that constant from being exposed in Swift, so I'm just going to create my own there for clarity.